src/core/Microsoft.PowerToys.Settings.Test/SettingsUtilsTests.cs

```diff
@@ -0,0 +1,108 @@
+using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerToys.Settings.UnitTest
+{
+    [TestClass]
+    public class SettingsUtilsTests
+    {
+        public SettingsUtilsTests()
+        {
+            string file_name = "\\test";
+            if (SettingsUtils.SettingsFolderExists(file_name))
+            {
+                DeleteFolder(file_name);
+            }
+        }
+        [TestMethod]
+        public void SaveSettings_SaveSettingsToFile_WhenFilePathExists()
+        {
+            // Arrange
+            string file_name = "\\test";
+            string file_contents_correct_json_content = "{\"name\":\"powertoy module name\",\"version\":\"powertoy version\"}";
+
+            BasePTSettingsTest expected_json = JsonSerializer.Deserialize<BasePTSettingsTest>(file_contents_correct_json_content);
+
+            // Act
+            SettingsUtils.SaveSettings(file_contents_correct_json_content, file_name);
```

Comment:
Similar to last comment.

Reply:
all folders are created under the \\test folder and this folder is deleted every time a test runs: https://github.com/microsoft/PowerToys/pull/2129/commits/1756d332fbe00938d366b0ea37147604660814bb

Reply:
I wasn't aware that a constructor will get called on every test execution

Reply:
Ran the changes locally looks the `test` folder left behind after the execution. Probably from the last test executing may we can use `TestCleanUp` [attribute to clean stuff ](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.testcleanupattribute?view=mstest-net-1.2.0)after test execution or`ClassCleanup`

Reply:
Added cleanup method: https://github.com/microsoft/PowerToys/pull/2129/commits/e415ae490a6da4807f1c33f5375d132ce0217fcc